Make it Great/SceneDelegate.swift

```diff
@@ -9,17 +9,18 @@ import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
+    //var viewController = MonitoringViewController()
 
     func scene(_ scene: UIScene, willConnectTo _: UISceneSession, options _: UIScene.ConnectionOptions) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
         guard let windowScene = (scene as? UIWindowScene) else { return }
-      
-      window = UIWindow(windowScene: windowScene)
 
-      window?.rootViewController = CustomNavigationController(rootViewController: MonitoringViewController())
-      window?.makeKeyAndVisible()
+        window = UIWindow(windowScene: windowScene)
+        UserModel().createUser(name: "Lucas", dayLastDrink: Date())
```

Comment:
Tem só que lembrar de tirar isso qnd for fazer a integração com o onboarding

Reply:
Blz